From 6928b8609db9b1c104c4cd4f9b163486121fb0f0 Mon Sep 17 00:00:00 2001
From: James Le Cuirot <chewi@gentoo.org>
Date: Sat, 8 Mar 2025 21:53:55 +0000
Subject: [PATCH] Fix building with no sndfile support at all

The stub functions still reference SNDFILE, so we still need to include
the bundled header in this case.
---
 source/decoder/sndfile_decoder.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/source/decoder/sndfile_decoder.h b/source/decoder/sndfile_decoder.h
index 2645ebd..3bf916a 100644
--- a/source/decoder/sndfile_decoder.h
+++ b/source/decoder/sndfile_decoder.h
@@ -43,6 +43,8 @@ private:
     static sf_count_t file_tell(void *user_data);
 };
 
+#else
+#include "../thirdparty/sndfile.h"
 #endif
 
 #endif /* SNDFILE_DECODER_H */
-- 
2.48.1

